baselines/dom.generated.d.ts

```diff
@@ -19025,6 +19024,7 @@ declare var console: Console;
 
 /** Holds useful CSS-related methods. No object with this interface are implemented: it contains only static methods and therefore is a utilitarian interface. */
 declare namespace CSS {
+    function escape(ident: string): string;
     function escape(ident: string): string;
```

Comment:
this duplicate `escape` is probably in addedTypes.json or overridingTypes.json, and should now be removed.

Reply:
LinkStyle is already included by this entry in idlSources, so removing the entry from my commit:  {
        "url": "https://drafts.csswg.org/cssom/",
        "title": "CSS Object Model"
    }
That was also the cause of the duplicate CSS.escapes